pennylane/collections.py

```diff
@@ -421,6 +422,58 @@ def qnode2(x, y):
 
     where the results from each QNode have been flattened and concatenated
     into a single one-dimensional list.
+
+    .. caution::
```

Comment:
The main reason for having two is so that both would stand out (both are equally important for the users to see). Having one large box would maybe 'drown out' the warnings more.